docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -81,6 +81,34 @@ Each Protocol simulation Unit carries out the following steps:
 
 Note: three different types of multistate sampling (i.e. replica swapping between lambda states) methods can be chosen; HREX, SAMS, and independent (no lambda swaps attempted). By default the HREX approach is selected, this can be altered using ``simulation_settings.sampler_method`` (default: ``repex``).
 
+Analysis
+~~~~~~~~
+
+As standard, some analysis of the each simulation repeat is performed.
+This analysis is made available through either the dictionary of results in the execution output,
+or through some ready-made plots for quick inspection.
+This analysis can be categorised as relating
+to the energetics of the different lambda states that were sampled,
+or to the analysis of the change in structural conformation over time in each state.
+
+* lambda energetic analysis
+
+  * free energy overlap matrix
```

Comment:
I think I would say MBAR overlap matrix or potential energy overlap matrix since this is not the overlap of the free energies, but potential energies, i think.